### PR TITLE
population numbers workaround

### PR DESCRIPTION
**Pull request by Z1Meme**

### CODE REVIEW

**Comment by Z1Meme** [comment]
@QuentinGruber This goes with a launcher update, do not merge without launchers updated first.

File: src/packets/LoginUdp/LoginUdp_11/loginpackets.ts [hunk 1]
```diff
@@ -23,7 +23,8 @@ const serverField: any[] = [
   { name: "descriptionId", type: "uint32" },
   { name: "reqFeatureId", type: "uint32" },
   { name: "serverInfo", type: "string" },
-  { name: "populationLevel", type: "uint32" },
+  { name: "populationNumber", type: "uint16", defaultValue: 0 },
```
Comment (QuentinGruber) [inline src/packets/LoginUdp/LoginUdp_11/loginpackets.ts:26]: Should put a comment about this explaining that we do modify this packet and that it's not is correct structure

File: src/servers/LoginServer/loginserver.ts [hunk 2]
```diff
@@ -226,20 +225,14 @@ export class LoginServer extends EventEmitter {
...
-                      .collection(DB_COLLECTIONS.SERVERS)
-                      .findOne({ serverId: serverId });
                     this._db
                       ?.collection(DB_COLLECTIONS.SERVERS)
                       .findOneAndUpdate(
                         { serverId: serverId },
                         {
                           $set: {
                             populationNumber: population,
-                            populationLevel: getPopulationLevel(
```
Comment (QuentinGruber) [inline src/servers/LoginServer/loginserver.ts:239]: if we remove it then remove the util func

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

**Comment by 0-x-2-2** [comment]
It could be possible to detect if this is an old client or not by changing the protocol version for custom protocol features supported or maybe just using a custom packet entirely.

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**